app/bundles/CampaignBundle/Tests/Executioner/RealTimeExecutionerTest.php

```diff
@@ -63,6 +64,8 @@ class RealTimeExecutionerTest extends \PHPUnit\Framework\TestCase
 
     private DecisionHelper $decisionHelper;
 
+    private EventRedirectionHelper $redirectionHelper;
```

Comment:
Isn't it Mock as well?
```suggestion
    private EventRedirectionHelper&MockObject $redirectionHelper;
```

Reply:
Fixed.